### DISCUSSION TITLE
Interval of updating dependencies

### DISCUSSION THREAD

**Discussion by sifferhans** [bcc-code/bmm-web#163] [General]
What is the preferred interval to update/bump dependencies? Should we have some routine for it?
I like to use a tool like [taze](https://github.com/antfu/taze) to stay on top of dependency updates.

Such updates should probably be done in separate PRs to keep things tidy, at least if there are lots of updated deps.

**Comment by SimonSimCity** [comment]
As long as we don't have a proper set of tests, I'd refrain from making this an automated routine. I'm checking it once in a while and often discover incompatibilities even in minor versions and bugfix versions.

Until we have a good set of tests, I'd keep it a manual process.

I've one update in the pipeline, and I'd prefer to do it either when needed/wanted (critical fix in a package, feature we want was added) or when some time passed. I feel like everything else is not viable at the current state of the project.
